app/views/contexts/summary/question.py

```diff
@@ -43,9 +48,31 @@ def _build_answers(self, answer_store, question_schema):
             if exclusive_option:
                 return summary_answers[-1:]
             return summary_answers[:-1]
-
         return summary_answers
 
+    def _concatenate_textfield_answers(self, answer_store, concatenation_type):
+
+        concatenated_answer = ""
+        answer_separators = {"NewLine": "<br/>", "Space": " "}
+
+        answer_separator = answer_separators.get(concatenation_type)
+
+        for answer_schema in self.answer_schemas:
+            answer_value = self._get_answer(answer_store, answer_schema["id"])
+
+            if answer_value:
+                if answer_separator and concatenated_answer:
+                    answer_value = f"{answer_separator}{answer_value}"
+                concatenated_answer += str(answer_value)
+
+        return [
+            {
+                "id": f"{self.id}-concatenated-answer",
+                "value": concatenated_answer,
+                "type": "textfield",
```

Comment:
- This should return a just the answer object, if it needs to be wrapped in a list then that should be done by the caller.
- I don't think `type` is needed here and should be removed.